### PR TITLE
Update defaultuserrolepermissions.md

### PR DESCRIPTION
**Pull request by purishd**
Updating the property and Json representation for Tenant creation lockdown feature

### CODE REVIEW

**Comment by purishd** [comment]
#label:"ready for content review"

File: api-reference/beta/resources/defaultuserrolepermissions.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ Contains certains customizable permissions of default user role in Azure AD.
 | allowedToCreateSecurityGroups | Boolean | Indicates whether the default user role can create security groups. |  
 | allowedToReadBitlockerKeysForOwnedDevice | Boolean | Indicates whether the registered owners of a device can read their own BitLocker recovery keys with default user role. |
 | allowedToReadOtherUsers | Boolean | Indicates whether the default user role can read other users. |  
+| allowedToCreateTenants | Boolean | Indicates whether the default user role can create tenants. |
```
Comment (isvargasmsft) [inline api-reference/beta/resources/defaultuserrolepermissions.md:24]: Missing in this PR: Changelog entry for the new property.

File: api-reference/beta/resources/defaultuserrolepermissions.md [hunk 2]
```diff
@@ -21,6 +21,7 @@ Contains certains customizable permissions of default user role in Azure AD.
...
```
Comment (isvargasmsft) [inline api-reference/beta/resources/defaultuserrolepermissions.md:25]: GENERAL: Is DougKirschner still the main point of contact for this article?

**Review by isvargasmsft — CHANGES_REQUESTED** [review]
Hello @purishd, 

Thank you very much for your work in this PR. 

I noticed you are on vacations, so I am just leaving a couple of comments by now. Please share with me the review documents on the side once you are back.

Kind regards, 

Isaac

**Comment by isvargasmsft** [comment]
Awating for review files (CSDL PR for beta endpoint).

**Comment by purishd** [comment]
@isvargasmsft I'm unsure of what changes have been requested. Can you please ping me on Teams about what I need to change on this?

File: api-reference/beta/resources/defaultuserrolepermissions.md (hunk 2 above)
Comment (isvargasmsft) [reply api-reference/beta/resources/defaultuserrolepermissions.md:25]: @purishd FYI

**Pull request closed without merging**